sophiread/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ set(CMAKE_CXX_STANDARD 14)
 
 # Dependencies
 find_package(Eigen3 REQUIRED)
-# find_package(OpenMP REQUIRED)
```

Comment:
Since we are using `tbb` to handle multi-threading, there is no need to use OpenMP anymore.

Reply:
ah, well I've actually got other work toward a larger PR to memory-mapped files, OpenMP and OpenACC.